src/pdsfile/holdings_maintenance/pds3/pdsinfoshelf.py

```
@@ -789,17 +790,17 @@ def main():
-        except (ValueError, IOError):
+        except (ValueError, OSError):
             # Allow a volume name to stand in for a .tar.gz archive
-            (dir, basename) = os.path.split(path)
-            pdsdir = pdsfile.Pds3File.from_abspath(dir)
+            (dirname, basename) = os.path.split(path)
+            pdsdir = pdsfile.Pds3File.from_abspath(dirname)
             if pdsdir.archives_ and '.' not in basename:
                 if pdsdir.voltype_ == 'volumes/':
                     basename += '.tar.gz'
                 else:
                     basename += '_%s.tar.gz' % pdsdir.voltype_[:-1]
```

Ruff (0.16.1)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)